src/components/ApyCalculatorModal/index.tsx

```diff
@@ -11,6 +11,7 @@ interface ApyCalculatorModalProps {
   linkLabel: string
   linkHref: string
   earningTokenSymbol?: string
+  roundingDecimals?: number
```

Comment:
Add options roundingDecimals parameter to the APR calculator helpers - necessary for tokens where $1000 worth will earn you less than 0.01 of that token per day. Logic for calculating the decimals is within `../PoolCard/AprRow.tsx`